diff --git a/config.def.h b/config.def.h
index 9efa774..3cd348a 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,6 +5,7 @@ static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
+static const char appmenuprompt[]   = "Applications";
 static const char *fonts[]          = { "monospace:size=10" };
 static const char dmenufont[]       = "monospace:size=10";
 static const char col_gray1[]       = "#222222";
@@ -57,7 +58,24 @@ static const Layout layouts[] = {
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static char dmenuborder[3];
+static char dmenuheight[3];
+static const char *dmenucmd[] = {
+	"dmenu_dex_run",
+	"-m", dmenumon,
+	"-fn", dmenufont,
+	"-i",
+	"-l", "20",
+	"-z", "480",
+	"-h", dmenuheight,
+	"-bw", dmenuborder,
+	"-p", appmenuprompt,
+	"-nb", col_gray1,
+	"-nf", col_gray3,
+	"-sb", col_cyan,
+	"-sf", col_gray4,
+	NULL
+};
 static const char *termcmd[]  = { "st", NULL };
 
 static const Key keys[] = {
diff --git a/dwm.c b/dwm.c
index 1443802..50ec9aa 100644
--- a/dwm.c
+++ b/dwm.c
@@ -1648,8 +1648,11 @@ spawn(const Arg *arg)
 {
 	struct sigaction sa;
 
-	if (arg->v == dmenucmd)
+	if (arg->v == dmenucmd) {
 		dmenumon[0] = '0' + selmon->num;
+		sprintf(dmenuheight, "%d", bh);
+		sprintf(dmenuborder, "%d", borderpx);
+	}
 	if (fork() == 0) {
 		if (dpy)
 			close(ConnectionNumber(dpy));
